### ISSUE TITLE
有考虑加个开机自动启动吗？

### ISSUE THREAD

**Issue by AoEiuV020** [jiangxufeng/v2rayL#60]
启动服务并添加任务栏图标，但是不打开ui界面的开机自启，

**Comment by jiangxufeng** [comment]
因为安装脚本里面设置了v2rayL.service文件的开机自启动，因此开机时会自动连接至上次连接的配置，程序的自启动目前是通过用户自行在`启动应用程序`里面添加命令实现的

**Comment by hustmzp** [comment]
> 因为安装脚本里面设置了v2rayL.service文件的开机自启动，因此开机时会自动连接至上次连接的配置，程序的自启动目前是通过用户自行在`启动应用程序`里面添加命令实现的

启动的命令是什么?

**Comment by AoEiuV020** [comment]
@hustmzp  说的是桌面提供的开机自启吧，不是什么命令，在设置之类的找找，
我用的cinnamon设置里有startup设置，
这种自启就和双击一样的效果，有窗口弹出来，

**Comment by hustmzp** [comment]
> @hustmzp 说的是桌面提供的开机自启吧，不是什么命令，在设置之类的找找，
> 我用的cinnamon设置里有startup设置，
> 这种自启就和双击一样的效果，有窗口弹出来，

是的,我需要的就是这个软件的开机自启,因为Ubuntu1804自带的startup application就可以设置开机启动项,然后找到了这个软件对应的命令,新建一个启动项在command栏填上`/usr/bin/v2rayL/v2rayLui`这条命令就可以了
